subsys/greybus/greybus_cport.c

```diff
@@ -43,11 +43,17 @@ enum {
 	};
 
 #define GB_PWM_PRIV_DATA(_node_id, _prop, _idx)                                                    \
-	static struct gb_pwm_channel_data gb_pwm_channel_data_arr_##_idx[1];                       \
+	static struct gb_pwm_channel_data gb_pwm_channel_data_arr_##_idx[                            \
```

Comment:
`DT_PROP_OR` already exists. So using `COND_CODE_1` unnecessary. Not that it matters, since everything else is also incorrect. Just something for future.